.travis.yml

```diff
@@ -27,6 +21,14 @@ install:
 script:
   - $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 npm run make
   - $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 npm test
+  # Test in several timezones (positive, negative, zero offset; DST; partial-hour offsets)
```

Comment:
the first of these will compile everything, and the remaining ones will all be quite fast.  I think build matrix would slow things down quite a bit with no real benefit.